src/hooks/useAuth.js

```diff
@@ -0,0 +1,49 @@
+import React, { useContext, useState, useEffect } from "react";
+import AuthContext, {
```

Comment:
AuthContext shouldn't be upper case

Reply:
do You mean it should look  like this ? : 
`const authContext = createContext(null);

<authContext.Provider
      value={{
        isAuthed,
        authenticate,
        logout,
        user
      }}
    >
      {children}
    </authContext.Provider>
`